### PR TITLE
Tiny enhancements for CI tests

### PR DESCRIPTION
**Pull request by zulinx86**
## Changes

- Show microcode version with `tools/devtool test`.
- Run the sysfs vulnerabilties test on host.

## Reason

See the commit messages.

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- ~~[ ] If a specific issue led to this PR, this PR closes the issue.~~
- [x] The description of changes is clear and encompassing.
- ~~[ ] Any required documentation changes (code and docs) are included in this PR.~~
- ~~[ ] API changes follow the [Runbook for Firecracker API changes][2].~~
- ~~[ ] User-facing changes are mentioned in `CHANGELOG.md`.~~
- [x] All added/changed functionality is tested.
- ~~[ ] New `TODO`s link to an issue.~~
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [x] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Review by roypat — APPROVED** [review]

**Review by kalyazin — APPROVED** [review]

File: tools/devtool [hunk 1]
```diff
@@ -556,6 +556,9 @@ cmd_test() {
 
     # If we got to here, we've got all we need to continue.
     say "Kernel version: $(uname -r)"
+    if [ "$(uname --machine)" = "x86_64" ]; then
+        say "Microcode version: $(sudo cat /sys/devices/system/cpu/cpu0/microcode/version)"
```
Comment (pb8o) [inline tools/devtool:560]: Isn't this included in the lscpu below?

I would be OK removing the lscpu call, as I usually have to scroll over it

**Review by pb8o — APPROVED** [review]

File: tools/devtool (hunk 1 above)
Comment (zulinx86) [reply tools/devtool:560]: > Isn't this included in the lscpu below?

I cannot see it in the `lscpu` result.

> I would be OK removing the lscpu call, as I usually have to scroll over it

I'm using the `lscpu` result to know if m5d.metal is skylake or cascade lake.

Comment (pb8o) [reply tools/devtool:560]: I could swear it was, but you are right. Thanks!

Comment (roypat) [inline tools/devtool:560]: Can we drop the `sudo` here (or more it into the "CI only" block below)? Its causing some inconvenience when running outside of our CI :(

Comment (zulinx86) [reply tools/devtool:560]: I didn't want to add the `sudo` there too. I saw an error of lack of access right somewhere, but I cannot find it now... Let me try to remove it in PR #4118

Comment (zulinx86) [reply tools/devtool:560]: https://buildkite.com/firecracker/firecracker-pr-optional/builds/2649#018ab343-b2c0-4ee9-b239-f665b4fc9531
```
cat: /sys/devices/system/cpu/cpu0/microcode/version: Permission denied
--
  | [Firecracker devtool 2023-09-20T15:50:11+0000] Microcode version:
```

Comment (zulinx86) [reply tools/devtool:560]: It appears to happen only on 4.14 kernel.

Comment (roypat) [reply tools/devtool:560]: Can we just move it to the "if [[ $BUILDKITE == "true ]];" block a few lines down? I don't think its very important to have this output locally (which is where it prompts me for a password now)

Comment (zulinx86) [reply tools/devtool:560]: Yes, we can, but I'd like to show it soon below the kernel version :(
Let me think another approach.

Comment (zulinx86) [reply tools/devtool:560]: /proc/cpuinfo solves this problem.

**Pull request merged**